irods/connection.py

```diff
@@ -454,16 +492,13 @@ def _login_pam(self):
         Pam_Long_Tokens = (ALLOW_PAM_LONG_TOKENS and (len(ctx) >= MAX_NAME_LEN))
 
         if Pam_Long_Tokens:
-
-            message_body = PamAuthRequest(
-                pamUser=self.account.client_user,
-                pamPassword=pam_password,
-                timeToLive=time_to_live_in_seconds)
+            message_body = PamAuthRequest( pamUser = self.account.client_user,
+                                           pamPassword = new_pam_password,
+                                           timeToLive = time_to_live_in_hours)
         else:
-
-            message_body = PluginAuthMessage(
-                auth_scheme_ = PAM_AUTH_SCHEME,
-                context_ = ctx)
+            # Divert to old 'PAM' scheme - TODO make issue, need client-side state
```

Comment:
TODO

Reply:
same question as above

Reply:
Yes, go ahead and create the issue and remove the TODO comment (or update with the new issue number).

Reply:
That comment didn't belong there anyway, I think.  Since the issue is made, we'll consider this done